content/en/docs/tasks/security/authz-tcp/index.md

```diff
@@ -91,72 +91,65 @@ to the `v2` version, which talks to a MongoDB backend using TCP.
 
 1. Verify that the **Book Reviews** section shows the reviews.
 
-## Enabling Istio authorization
+## Applying a default `deny-all` policy
 
-Run the following command to enable Istio authorization for the MongoDB service:
+Run the following command to apply a default `deny-all` policy for the MongoDB workload:
 
 {{< text bash >}}
-$ kubectl apply -f @samples/bookinfo/platform/kube/rbac/rbac-config-on-mongodb.yaml@
+$ kubectl apply -f - <<EOF
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: deny-all
+spec:
+  selector:
+    matchLabels:
+      app: mongodb
+EOF
 {{< /text >}}
 
+The command creates a `deny-all` policy that selects the MongoDB workload and will deny all requests because
```

Comment:
Add the information before the command to ensure readers know what will happen before they run the command.
This avoids readers exposing or restricting workloads without knowing they did so.
It is not clear from the text what are the denied requests. Those coming from the MongoDB workload or those going to the MongoDB workload. Please rewrite to ensure that information is clear.